tests/jvm/src/test/scala-2.12/scala/meta/tests/semanticdb/IncrementalSuite.scala

```diff
@@ -35,6 +35,7 @@ class IncrementalSuite extends FunSuite with BeforeAndAfterEach with DiffAsserti
        |
        |Toplevels:
        |==========
+       |_empty_. => src/A.scala.semanticdb
```

Comment:
Note, these symbols will be removed after #1647

Reply:
Sounds good.